src/Files/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -225,6 +225,18 @@
                         <FontIcon Glyph="&#xE756;" />
                     </AppBarButton.Icon>
                 </AppBarButton>
+                <AppBarButton
+                    x:Name="SetAsBackgroundButton"
+                    x:Load="{x:Bind ViewModel.IsImage, Mode=OneWay, FallbackValue=False}"
+                    AutomationProperties.Name="SetAsBackground"
```

Comment:
The automation name will automatically be picked up in this case since we already set the label. I usually set the name if there isn't any text or if the text alone wouldn't provide enough context.

Reply:
Ah okay, would you like to remove it?

Reply:
Please